### DISCUSSION TITLE
How to revalidate/regenerate without triggering another build?

### DISCUSSION THREAD

**Discussion by jahzlariosa** [colbyfayock/next-wordpress-starter#434] [Q&A] [answered]
When updating content is there any way not to trigger another CI build? I had to always trigger a new build to get my updates loaded.

**Comment by jahzlariosa** [comment, marked as answer]
Never mind, I should have researched first :D 

adding revalidate helped :D

**Reply by colbyfayock** [reply]
for those following along, was your solution to _not_ trigger any updates on content change and only use revalidate?

## Revalidate / ISR Resources

https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration
https://www.youtube.com/watch?v=_isECpf3-8s

**Reply by jahzlariosa** [reply]
Yes this is it @colbyfayock. Thank you.
